modules/dedicated-cluster-install-deploy.adoc

```diff
@@ -19,19 +19,27 @@ Cluster Logging Operator and Elasticsearch Operator.
 
 .Prerequisites
 
-Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node
-requires its own storage volume.
+You are provided with a maximum of 600GiB of persistent storage exclusively
+available for cluster logging related Elasticsearch deployments. Note that each
+Elasticsearch node requires its own storage volume.
 
-Elasticsearch is a memory-intensive application. Each Elasticsearch node needs 16G of memory for both memory requests and limits.
-The initial set of {product-title} nodes might not be large enough to support the Elasticsearch cluster. You must add additional nodes to the
-{product-title} cluster to run with the recommended or higher memory. Each Elasticsearch node can operate with a lower
-memory setting though this is not recommended for production deployments.
+Elasticsearch is a memory-intensive application. Each Elasticsearch node needs
+8G of memory for both memory requests and limits. Each Elasticsearch node can
+operate with a lower memory setting though this is not recommended for
+production deployments.
 
 .Procedure
 
-. Install the Elasticsearch Operator via the OperatorHub. See: xref:../operators/olm-adding-operators-to-cluster.adoc[Adding Operators to a cluster].
+. Install the Elasticsearch Operator via the OperatorHub:
 
-. Install the Cluster Logging Operator using the {product-title} web console for best results:
+.. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+.. Choose  *Elasticsearch* from the list of available Operators, and click *Install*.
+
+.. On the *Create Operator Subscription* page, under *A specific namespace on the cluster* select *openshift-logging*.
```

Comment:
Does this namespace exist by default? I don't see instructions to make it in the assembly.

Reply:
Correct - great question!